### PR TITLE
[docs] Clarify benefits of colocated tables

### PR DESCRIPTION
**Pull request by ddhodge**
Edit for clarity of tradeoffs of using colocated tables

@netlify /preview/architecture/docdb-sharding/colocated-tables

### CODE REVIEW

**Review by lonedevel — APPROVED** [review]
Looks great!

File: docs/content/preview/architecture/docdb-sharding/colocated-tables.md [hunk 1]
```diff
@@ -65,20 +65,22 @@ This is characteristic of a microservices-oriented architecture, where each micr
 
 **Example:** Multi-tenant SaaS services where one database is created per customer. As new customers are rapidly on-boarded, it becomes necessary to add more databases quickly while maintaining high-availability and fault-tolerance of each database.
```
Comment (lizayugabyte) [inline docs/content/preview/architecture/docdb-sharding/colocated-tables.md:66]: Under "Motivation" at the top of the page:

This feature is desirable in a number of scenarios, some of which are described below. --> Should be rephrased in general and "below" shouldn't be used.

File: docs/content/preview/architecture/docdb-sharding/colocated-tables.md [hunk 2]
```diff
@@ -65,20 +65,22 @@ This is characteristic of a microservices-oriented architecture, where each micr
...
 
-## Tradeoffs
+## Benefits and tradeoffs
 
-Fundamentally, colocated tables have the following tradeoffs:
+Colocated tables have the following benefits:
 
 - **Higher performance - no network reads for joins**.
```
Comment (lizayugabyte) [inline docs/content/preview/architecture/docdb-sharding/colocated-tables.md:72]: Why is this in bold?

File: docs/content/preview/architecture/docdb-sharding/colocated-tables.md [hunk 3]
```diff
@@ -65,20 +65,22 @@ This is characteristic of a microservices-oriented architecture, where each micr
...
-All of the data across the various colocated tables is local, which means joins no longer have to
+
+  All of the data across the various colocated tables is local, which means joins no longer have to
 read data over the network. This improves the speed of joins.
 
 - **Support higher number of tables - using fewer tablets**.
```
Comment (lizayugabyte) [inline docs/content/preview/architecture/docdb-sharding/colocated-tables.md:77]: Why is this in bold?

File: docs/content/preview/architecture/docdb-sharding/colocated-tables.md [hunk 4]
```diff
@@ -65,20 +65,22 @@ This is characteristic of a microservices-oriented architecture, where each micr
...
-Because multiple tables and indexes can share one underlying tablet, a much higher number of tables
-can be supported using colocated tables.
 
-- **Lower scalability - until removal from colocation tablet**.
-The assumptions behind tables that are colocated is that their data need not be automatically sharded and distributed across nodes. If it is known a priori that a table will get large, it can be opted out of the colocation tablet at creation time. If a table already present in the colocation tablet gets too large, it can dynamically be removed from the colocation tablet to enable splitting it into multiple tablets, allowing it to scale across nodes.
+  Because multiple tables and indexes can share one underlying tablet, a much higher number of tables can be supported using colocated tables.
+
+However, these come with the tradeoff of **lower scalability** for large tables, until the tables are removed from the colocation tablet.
```
Comment (lizayugabyte) [inline docs/content/preview/architecture/docdb-sharding/colocated-tables.md:81]: Why is lower scalability in bold?

(bold is usually reserved for referring to UI elements)

**Review by lizayugabyte — APPROVED** [review]
Minor comments

**Pull request merged**